src/pg-storage-client.ts

```diff
@@ -31,7 +31,8 @@ export default class PGStorageClient {
         'bot_counters': 'CREATE TABLE bot_counters (user_id BIGINT PRIMARY KEY, counter INTEGER);',
         'privileged_roles': 'CREATE TABLE privileged_roles (guild_id BIGINT PRIMARY KEY, role_id BIGINT);',
         'daily_analytics': 'CREATE TABLE daily_analytics (date DATE, label SMALLINT, value INTEGER, PRIMARY KEY (date, label));',
-        'misc_properties': 'CREATE TABLE misc_properties (name VARCHAR(32) PRIMARY KEY, value VARCHAR(2048));'
+        'misc_properties': 'CREATE TABLE misc_properties (name VARCHAR(32) PRIMARY KEY, value VARCHAR(2048));',
+        'guild_settings': 'CREATE TABLE guild_settings (guild_id BIGINT, setting SMALLINT, value SMALLINT, PRIMARY KEY (guild_id, setting));'
```

Comment:
tismnit: move `guild_settings` to above `misc_properties` and `daily_analytics` since those are both metadata-ish